packages/bridge-ui/src/components/form/BridgeForm.svelte

```diff
@@ -422,7 +422,7 @@
     on:click={approve}
     disabled={btnDisabled}
   >
-    {$_("home.approve")}
+    {$_("home.bridge")}
```

Comment:
We already have a `bridge` button at https://github.com/taikoxyz/taiko-mono/pull/13163/files#diff-59da5293d790c811f91dfac83d947df350a849a18030f5b5399dadb4da5ce3c8R417

To fix the issue mentioned we need some change to the overall rendering logic, just changing the text here doesn't solve the problem.